test/factories/user.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :user do
+    github_id 1234
+    access_token '54c543ffd45343fg5434fgg556ff553232gg57890v'
```

Comment:
Since there are unique indexes on `github_id` and `access_token` columns I'm pretty sure the linting for FactoryGirl will fail for the user.